samples/fabric/fabric-cli/src/commands/asset/exchange/lock.ts

```diff
@@ -194,8 +194,9 @@ const command: GluegunCommand = {
         spinner.fail(`Error`)
     }
 
-    await network.gateway.disconnect()
+    // await new Promise(f => setTimeout(f, options['timeout-duration'] * 1000 + 3000));
```

Comment:
Delete this line.

Reply:
We wanted to retain this commented line, incase someone wants to test automated unlock, they will have to use this to sleep till timeout expires, so that the process doesn't get killed.